brian2/stateupdaters/base.py

```diff
@@ -127,10 +138,16 @@ def apply_stateupdater(equations, variables, method):
             if the_method is None:
                 raise ValueError(('No stateupdater that is suitable for the '
                                   'given equations has been found.'))
-            logger.info('Using state updater: %r' % the_method)
+            if group_name is not None:
+                msg_text = ("No numerical integration method specified for group "
+                            "'{group_name}', choosing method '{method}'.")
+            else:
+                msg_text = ("No numerical integration method specified, "
+                            "choosing method '{method}'.")
+            logger.info(msg_text.format(group_name=group_name,
+                                        method=the_method), 'method_choice')
```

Comment:
Hmm, no I was thinking about moving all of the timing information to the `DEBUG` log -- the thing is that the time that the method itself took is not necessarily interesting in the cases with complicated equations where checking the methods takes a long time but then applying Euler is quick. But maybe adding both times to the `INFO` log is ok actually.